packages/remix-eslint-config/rules/typescript.js

```diff
@@ -1,54 +1,49 @@
-const OFF = 0;
-const WARN = 1;
-const ERROR = 2;
-
 module.exports = {
-  "no-dupe-class-members": OFF,
-  "no-undef": OFF,
-
-  // Add TypeScript specific rules (and turn off ESLint equivalents)
-  "@typescript-eslint/consistent-type-assertions": WARN,
-  "@typescript-eslint/consistent-type-imports": WARN,
```

Comment:
Per https://typescript-eslint.io/rules/, these aren't enabled in `@typescript-eslint/recommended` so should we be keeping them?

Reply:
They can be kept if desired, sure. We removed it from the _recommended_ list because they're really a _stylistic_ concern. In the upcoming v6 of typescript-eslint they're added to the _stylistic_ config: https://v6--typescript-eslint.netlify.app/rules/consistent-type-assertions -> https://v6--typescript-eslint.netlify.app/linting/configs/#stylistic

What we could do is wait until typescript-eslint v6 is published hopefully in a few weeks _(which is something I've been saying for months)_, then extend both the _recommended_ and _stylistic_ configs. Thoughts?

Reply:
ok, going to keep them in for now with a note that they can be removed once v6 is released and we adopt `@typescript-eslint/stylistic`.  I'm push up some changes shortly